packages/admin-next/dashboard/src/routes/products/common/upload-media-form-item.tsx

```diff
@@ -0,0 +1,87 @@
+import { useTranslation } from "react-i18next"
+import { FileType, FileUpload } from "../../../components/common/file-upload"
+import { Form } from "../../../components/common/form"
+
+const SUPPORTED_FORMATS = [
+  "image/jpeg",
+  "image/png",
+  "image/gif",
+  "image/webp",
+  "image/heic",
+  "image/svg+xml",
+]
+
+const SUPPORTED_FORMATS_FILE_EXTENSIONS = [
+  ".jpeg",
+  ".png",
+  ".gif",
+  ".webp",
+  ".heic",
+  ".svg",
+]
+
+export const UploadMediaFormItem = ({
+  form,
+  append,
+}: {
+  form: any
+  append: any
+}) => {
+  const { t } = useTranslation()
+
+  const hasInvalidFiles = (fileList: FileType[]) => {
+    const invalidFile = fileList.find(
+      (f) => !SUPPORTED_FORMATS.includes(f.file.type)
+    )
```

Comment:
suggestion: maybe we should use a filter to display all of them to the user in order for the user not to change them one by one and having an error for them each time

Reply:
@adrien2p yeah i think they will be shown in the upload component, but that whole UI needs to be revisited